uai2017/python/match_utils.py

```diff
@@ -1,3 +1,13 @@
+import pickle
+
+def save_obj(obj, name):
+    with open(name + '.pkl', 'wb') as f:
+        pickle.dump(obj, f, pickle.HIGHEST_PROTOCOL)
```

Comment:
I'm leaving the .pkl extension in the save_obj function here because the user shouldn't be able to save a non-pkl file.

unfortunately this means that save/load are asymmetrical, which is strange.

is this an OK way to do it? on the one hand, it's nice to pass in autocompleted full filenames, but on the other hand, we don't want people trying to save non-pkl files